README.md

```diff
@@ -9,12 +9,12 @@ This packaged was designed to be used with the template files generated by
 for `.po`/`.pot` file checking.
 
 ## Getting Started
-This plugin requires Grunt `~0.4.2`
+This plugin requires Grunt `>=0.4.0`
 
 If you haven't used [Grunt](http://gruntjs.com/) before, be sure to check out the [Getting Started](http://gruntjs.com/getting-started) guide, as it explains how to create a [Gruntfile](http://gruntjs.com/sample-gruntfile) as well as install and use Grunt plugins. Once you're familiar with that process, you may install this plugin with this command:
 
 ```shell
-npm install git://github.com/incuna/grunt-gettext-checker.git#2.0.0 --save-dev
+npm install git://github.com/incuna/grunt-gettext-checker.git#3.0.1 --save-dev
```

Comment:
Yes, since this is published on npm. But wanted to keep this change simple for now